.yamato/project-pack.yml

```diff
@@ -0,0 +1,17 @@
+{% metadata_file .yamato/project.metafile %}
```

Comment:
what's the use case for this packing?

Reply:
To run upm project tests, you run this project pack command first. See [here](https://github.cds.internal.unity3d.com/unity/upm-ci-utils#2-upm-ci-project-test).